CONTRIBUTING.md

```diff
@@ -7,6 +7,14 @@ Thanks for helping out!
 
 Run `npm run demo` to run a webpack dev server with component examples.
 
+## Generator Specific
+
+All boilerplate components should be named `boilerplate-component` or `BoilerplateComponent` as appropriate.
+
+`src/components/boilerplate-component.jsx` SHOULD NOT be moved or renamed
+
+The dev dependency `replace` is required by the generator.
```

Comment:
no unfortunately, anything that is done on the *generated* project relies on dependencies of that project.  Yeoman is weird.